int/hwsc-file-transaction-svc/hwsc-file-transaction-svc.js

```diff
@@ -24,50 +24,67 @@ function getStatus(callback) {
   const request = {};
   client.getStatus(request, (err, response) => {
     if (!err) {
-      grpc.closeClient(client);
-    }
-    callback(err, response);
-  });
+    grpc.closeClient(client);
+  }
+  callback(err, response);
+});
 }
 
-function uploadFile(filePath, fileName, callback) {
+function createUserFolder(uuid, callback) {
   if (typeof callback !== 'function') {
     console.error('callback not a function');
     return;
   }
+  const id = uuid;
+  console.log(id);
+  const request = {};
+  request.uuid = uuid;
+  const server = client.createUserFolder(request, (err, response) => {
+    if (!err) {
+    grpc.closeClient(client);
+  }
+  callback(err, response);
+});
```

Comment:
`client.createUserFolder` block needs to be indented one more space, or you can eslint it